src/test/skript/tests/syntaxes/expressions/ExprCommandBlockCommand.sk

```diff
@@ -0,0 +1,32 @@
+test "command block command":
+	set block at location(0,1,0,world) to command block
+	spawn a minecart with command block at (spawn of world "world"):
+		set {_e} to entity
+	spawn a zombie at (spawn of world "world"):
+		set {_z} to entity
+	set {_b} to block at location(0,1,0,world)
+
+	assert command block command of {_b} is not set with "a new command block shouldn't have an associated command"
+	assert command block command of {_e} is not set with "a new minecart with command block shouldn't have an associated command"
+
+	set command block command of {_b} to "say asdf"
+	set command block command of {_e} to "say asdf"
+	assert command block command of {_b} is "say asdf" with "setting the command of a command block should do exactly that"
+	assert command block command of {_e} is "say asdf" with "setting the command of a minecart with command block should do exactly that"
+
+	clear command block command of {_b}
+	clear command block command of {_e}
+	assert command block command of {_b} is not set with "clearing/deleting/resetting the command of a command block should return null"
+	assert command block command of {_e} is not set with "clearing/deleting/resetting the command of a minecart with command block should return null"
+
+	set block at {_b} to stone
+	assert command block command of {_b} is not set with "the command of a non-command block should return null"
+	assert command block command of {_z} is not set with "the command of a non-minecart with command block should return null"
+
+	set command block command of {_b} to "say asdf"
+	set command block command of {_z} to "say asdf"
+	assert command block command of {_b} is not set with "setting the command of a non-command block shouldn't do anything"
+	assert command block command of {_z} is not set with "setting the command of a non-minecart with command block shouldn't do anything"
```

Comment:
Some tests with just `command of x` would be good to catch possible future syntax collisions.